src/axom/core/Path.cpp

```diff
@@ -45,12 +62,29 @@ Path Path::join(std::initializer_list<Path> paths, const char delim)
               path.m_components.end(),
               std::back_inserter(result.m_components));
   }
+
+  if(paths.size() != 0 && (*paths.begin()).m_leading_delim)
+  {
+    result.m_leading_delim = true;
+  }
+  else
+  {
+    result.m_leading_delim = false;
+  }
+
   return result;
 }
 
 Path::operator std::string() const
 {
-  return fmt::format("{0}", fmt::join(m_components, std::string(1, m_delim)));
+  std::string result;
+  if(m_leading_delim)
+  {
+    result = "/";
+  }
+
+  result += fmt::format("{0}", fmt::join(m_components, std::string(1, m_delim)));
```

Comment:
Suggestion: The following one-liner could account for leading delims and joining the path
```suggestion
  return fmt::format("{0}{1}", 
                 m_leading_delim? "" : std::string(1,m_delim), 
                 fmt::join(m_components, std::string(1, m_delim)));
```

Reply:
Thanks, I have put this in.  I needed to reverse the order of the second and third operands in the ternary operator.